### PR TITLE
Sync browser storage with vuex

### PR DESCRIPTION
**Pull request by petbaik**
I'm not sure if I covered all cases, but tests are passing (except send tip). I think we should test it and when we are sure that everything is ok then merge it.

### CODE REVIEW

**Comment by vmangelovv** [comment]
On first open after build, it throws errors:
image
Оtherwise everything looks good.

**Comment by mradkov** [comment]
@petbaik can you resolve conflicts?

**Comment by petbaik** [comment]
@davidyuk Can you check if `persistState` plugin is integrated properly

**Comment by mradkov** [comment]
@petbaik can you resolve conflicts?

File: src/store/plugins/persistState.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
+import { cloneDeep } from 'lodash-es';
+
 const KEY = 'state';
 
 const setState = async state => {
-  browser.storage.local.set({ [KEY]: state });
+  browser.storage.local.set({ [KEY]: cloneDeep(state) });
```
Comment (davidyuk) [inline src/store/plugins/persistState.js:6]: what was the problem there?

Comment (petbaik) [reply src/store/plugins/persistState.js:6]: There was problem with saving the state in Firefox and `cloneDeep` fixed the problem. Not sure what was the exact error.

Comment (davidyuk) [reply src/store/plugins/persistState.js:6]: Putting to local storage is already a kind of cloning, this part is super unobvious.

**Pull request merged**